### PR TITLE
Samples updated to latest cosmos sdk client

### PR DESCRIPTION
**Pull request by simplynaveen20**
Samples updated to latest cosmos sdk client.

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Resource/CosmosExtensions.cs [hunk 1]
```diff
@@ -52,7 +53,11 @@ internal static string ToSqlQueryText<T>(this IQueryable<T> query)
         /// </example>
         public static FeedIterator<T> ToFeedIterator<T>(this IQueryable<T> query)
         {
-            return ((CosmosLinqQuery<T>)query).ToFeedIterator();
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/Resource/CosmosExtensions.cs]: Not related to this PR.

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosItemTests.cs [hunk 2]
```diff
@@ -1195,6 +1195,18 @@ public async Task ItemLINQQueryTest()
                 Assert.AreEqual(2, resultsFetched);
             }
 
+            //Checking for exception in case of ToFeedIterator() use on non cosmos linq IQueryable.
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosItemTests.cs]: Not related to this PR.

File: Microsoft.Azure.Cosmos/src/Resource/CosmosExtensions.cs (hunk 1 above)
Comment (simplynaveen20) [reply Microsoft.Azure.Cosmos/src/Resource/CosmosExtensions.cs]: By bad , went it due to two parallel PR

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosItemTests.cs (hunk 2 above)
Comment (simplynaveen20) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/CosmosItemTests.cs]: By bad , went it due to two parallel PR

**Comment by kirankumarkolli** [comment]
Moving forward and will address the gate later.

**Pull request merged**